src/style.css

```diff
@@ -1,3 +1,197 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  margin: auto;
+  margin-top: 25px;
+  width: 198px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+
+  padding-top: 32px;
+  padding-left: 19px;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+.card__text {
+  width: 166px;
+  height: auto;
+
+  margin-top: 40px;
+  margin-left: 16px;
+  
+  color: var(--Main-accent, #060b35);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px; /* 150% */
+}
+
+.card__productCode {
+  width: 100px;
+  height: auto;
+
+  margin-top: 4px;
+  margin-left: 16px;
+
+  color: var(--Secondary, #616070);
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px; /* 140% */
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  width: 166px;
+  height: auto;
+
+  margin-top: 16px;
+  margin-left: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__reviewNumber {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  flex-shrink: 0;
+
+  width: 53px;
+  height: 16px;
+
+  color: var(--Main-accent, #060b35);
+  font-size: 10px;
+  font-style: normal;
+  text-align: right;
+  font-weight: 400;
+  line-height: 14px; /* 140% */
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  width: 166px;
+  height: 18px;
+
+  margin-left: 16px;
+  margin-top: 24px;
+}
+
+.card__priceText {
+  color: var(--Secondary, #616070);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px; /* 150% */
+  margin: 0;
+}
+
+.card__priceNumber {
+  width: 50px;
```

Comment:
```suggestion

```

Reply:
<img width="622" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/f5ea488c-f109-4155-9c52-5942a4bbf932">
still not fixed